README.md

```diff
@@ -35,64 +43,71 @@ Create/retrieve key:
     }
     ...
 
-    if err, key2 := tags.CreateKeyString("keyNameID2"); err != nil {
-        // handle error
-    }
-    ...
 
-### Create a set of tags associated with keys
-To create a new tag set from scratch using changes:
+### To create a set of tags associated with keys
+The following code demonstrates how to create a new tag set from scratch:
 
     tsb := NewTagSetBuilder(nil)
     tsb.InsertString(key1, "foo value")
     tsb.UpdateString(key1, "foo value2")
     tsb.UpsertString(key2, "bar value")
     tagsSet := tsb.Build()
 
-    // A shorter way of achieving the same is:
+The methods on builders can be chained and the following is a shorter version of achieving the same:
```

Comment:
Looks like the same code. Can we just have one or the other? Can be addressed in a different CL.

Reply:
Just removed